brave/brave-core
# NTP Widget Settings design updates

Resolves brave/brave-browser#10318

Note: The removal of "Show" text has been moved to a separate issue/PR https://github.com/brave/brave-browser/issues/10591

![Screen Shot 2020-07-06 at 9 41 37 PM](https://user-images.githubusercontent.com/8732757/86713284-c5c0b680-bfd2-11ea-8f88-07931a0d6c34.png)

![Screen Shot 2020-07-06 at 8 30 19 PM](https://user-images.githubusercontent.com/8732757/86713299-c9ecd400-bfd2-11ea-9db1-892c6938214c.png)

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
Additional specifications defined in issue for style updates. Functional testing plan with new "Add Card tab"
1. Build Brave with a clean profile
2. Navigate to `brave://newTab`
3. Hover over the "Add Card" widget, ensure the ellipsis menu is not available.
4. Remove the Binance and Rewards widget from the stack
5. Hover over the "Add Card" widget, ensure the menu is available, hide the card.
6. Under `More Cards` in the NTP settings, add back either widget.
7. Ensure the "Add Card" widget appears back, at the very top.

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
